packages/api/src/beacon/routes/events.ts

```diff
@@ -35,6 +35,10 @@ export enum EventType {
   attestation = "attestation",
   /** The node has received a valid voluntary exit (from P2P or API) */
   voluntaryExit = "voluntary_exit",
+  /** The node has received a ProposerSlashing (from P2P or API) that passes validation rules of the `proposer_slashing` topic */
+  proposerSlashing = "proposer_slashing",
+  /** The node has received an AttesterSlashing (from P2P or API) that passes validation rules of the `attester_slashing` topic */
+  attesterSlashing = "attester_slashing",
```

Comment:
I'd suggest to align this with existing documentation

```suggestion
  /** The node has received a valid proposer slashing (from P2P or API) */
  proposerSlashing = "proposer_slashing",
  /** The node has received a valid attester slashing (from P2P or API) */
  attesterSlashing = "attester_slashing",
```